auth/builtin.go

```diff
@@ -59,13 +59,28 @@ func (r *NoAuth) Refresh(refresh string) (token Token, err error) {
 }
 
 // Builtin auth provider.
-type Builtin struct {
+type Builtin struct{}
+
+// Parse Token out of a string
+func ParseToken(requestToken string) (string, error) {
```

Comment:
Why is this an exported package function instead of a non-exported method on Builtin?